components/nav.js

```diff
@@ -1,6 +1,6 @@
+// import Logo from './logo';
```

Comment:
For some reason when deploying webpack cannot resolve this component. Ideas how to solve it?